index.d.ts

```diff
@@ -524,15 +524,20 @@ declare namespace Moleculer {
 		[name: string]: any;
 	}
 
-	type ServiceEventHandler = ((payload: any, sender: string, eventName: string) => void) & ThisType<Service>;
+	type ServiceEventOldHandler = ((payload: any, sender: string, eventName: string, ctx: Context) => void) & ThisType<Service>;
```

Comment:
In keeping with how the below 0.14 signature is documented as "Legacy", I would name this type `ServiceEventLegacyHandler` to avoid any potential confusion.